src/commands/backup/schedule/create.tsx

```diff
@@ -0,0 +1,100 @@
+import { ExecRenderBaseCommand } from "../../../lib/basecommands/ExecRenderBaseCommand.js";
+import {
+  makeProcessRenderer,
+  processFlags,
+} from "../../../rendering/process/process_flags.js";
+import { ReactNode } from "react";
+import { Flags } from "@oclif/core";
+import { assertStatus } from "@mittwald/api-client-commons";
+import { Success } from "../../../rendering/react/components/Success.js";
+import { Value } from "../../../rendering/react/components/Value.js";
+
+import { projectFlags } from "../../../lib/resources/project/flags.js";
+
+type Result = {
+  projectBackupScheduleId: string;
+};
+
+export class Create extends ExecRenderBaseCommand<typeof Create, Result> {
+  static summary = "Create a new backup schedule";
+  static flags = {
+    ...projectFlags,
+    ...processFlags,
+    description: Flags.string({
+      description: "Set the description for the backup schedule",
+    }),
+    schedule: Flags.string({
+      required: true,
+      description: "Define the schedule itself",
+    }),
+    ttl: Flags.string({
+      required: true,
+      description:
+        "Define the backup storage period in days, for through this schedule created backups",
+    }),
+  };
+
+  protected async exec(): Promise<Result> {
+    const process = makeProcessRenderer(this.flags, "Creating a new sftp User");
+    const projectId = await this.withProjectId(Create);
+    const { description, schedule, ttl } = this.flags;
+
+    const createBackupSchedulePayload: {
+      description?: string | undefined;
+      schedule: string;
+      ttl: string;
+    } = {
+      schedule,
+      ttl,
+    };
+
+    if (description) {
+      createBackupSchedulePayload.description = description;
+    }
```

Comment:
If `description` is of type `string|undefined` anyway, this could be simplified to:

```suggestion
    const createBackupSchedulePayload: {
      description?: string | undefined;
      schedule: string;
      ttl: string;
    } = {
      description,
      schedule,
      ttl,
    };
```

Reply:
I was under the impression that if I PATCH a parameter as "undefined" for an existing Parameter, it will get purged.
Meaning if it doesn't contain a value but the property is set in the Studio, after the Request its gone.
That's why I seperated optional parameters - so that they aren't a part of the update payload unless they are set.

If my assumprion is wrong I'll gladly simplify the code. I just want to make sure.